### PR TITLE
fix: android dextrose

### PR DESCRIPTION
**Pull request by kaliabadi**

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/d4cdfcd364d429d38bf748bf74648966c396c232/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/4590f012cf1f64581767a6208767bf12b3084367/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/55a85bbdae29c8464b95cb2a744dc2674dc75e35/dextrose-test/index.html

File: android/gradle.properties [hunk 1]
```diff
@@ -22,3 +22,4 @@ MYAPP_RELEASE_STORE_FILE=my-release-key.keystore
 MYAPP_RELEASE_KEY_ALIAS=my-key-alias
 MYAPP_RELEASE_STORE_PASSWORD=tools123
 MYAPP_RELEASE_KEY_PASSWORD=tools123
+android.enableAapt2=false
```
Comment (fourlastor) [inline android/gradle.properties]: missing newline at end of file

File: dextrose/dextrose.android.js [hunk 2]
```diff
@@ -3,11 +3,11 @@ const path = require("path");
 const config = {
   snapPath: path.join(__dirname, "/snappy"),
   platformName: "Android",
-  platformVersion: "7.1.1",
-  deviceName: "Nexus_5X_API_25",
+  platformVersion: "5.1",
```
Comment (fourlastor) [inline dextrose/dextrose.android.js]: how come this has been changed to a previous version of android?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/fa092bd2e5dd7b759917dee7966bb3623831e4eb/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/8ea46124c3e49516200f3337f3f4de5c74c54673/dextrose-test/index.html

File: lib/visual_snapshots/run-android-dextrose.sh [hunk 3]
```diff
@@ -2,7 +2,7 @@
 npx appium &
 APPIUM_PID=$!
 LOCAL=true npx fructose-tunnel
-emulator @Nexus_5X_API_25 -no-boot-anim &
+/Users/kourosaliabadi/Library/Android/sdk/tools/emulator @Nexus_5X_API_25 -no-boot-anim &
```
Comment (L0wry) [inline lib/visual_snapshots/run-android-dextrose.sh]: 🥇

Comment (kaliabadi) [reply lib/visual_snapshots/run-android-dextrose.sh]: 🤦‍♂️ everytime

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/44e7b0e47265e30b75a1b6294f39c8f5840fbb10/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/0429ca2793416cd800020cd02f5a9b0fc2b6dcaa/dextrose-test/index.html

**Pull request merged**